src/main/webapp/gast/mghlemma.jsp

```diff
@@ -10,11 +11,35 @@
 <%@ page import="com.lowagie.text.*" isThreadSafe="false"%>
 <%@ page import="com.lowagie.text.rtf.*" isThreadSafe="false"%>
 <%@ page import="java.io.*" isThreadSafe="false"%>
+<%@ page import="de.uni_tuebingen.ub.nppm.exception.*" isThreadSafe="false" %>
 <jsp:include page="../dofilter.jsp" />
 
 <%
     int id = Integer.parseInt(request.getParameter("ID"));
 
+    if(MghLemmaDB.getById(id) == null){
+        throw new IdNotFoundException("Lemma ID ist nicht vorhanden");
+    }
+
+    List<Einzelbeleg> listEinzelbeleg = EinzelbelegDB.getListEinzelbelegeByMghLemmaId(id);
```

Comment:
Hier sollten wir auch eher eine Funktion in MghLemmaDB verwenden um ein MghLemma objekt über die ID abzufragen und dann darauf getEinzelbelege() aufrufen.

Reply:
done